contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,153 @@
+---
+date: 2024-05-28
+title: What we built at our Mykonos hackathon
```

Comment:
I personally like have an extra descriptor here - like how we had 'sun-kissed' for Aruba.

Reply:
wind swept?

Reply:
how romantic!